.github/workflows/sdk-generation.yaml

```diff
@@ -33,6 +33,21 @@ on:
             - https://example.com/openapi2.json
         required: false
         type: string
+      overlay_docs:
```

Comment:
@ThomasRooney you add an `overlay_docs` input to the action which is a yaml string representing an array of overlay files